libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -1,20 +1,20 @@
 <ng-container *ngIf="(readingList$ | async).length > 0; else empty">
-  <div class="reading-list-item" *ngFor="let b of readingList$ | async">
+  <div class="reading-list-item" *ngFor="let book of readingList$ | async">
     <div>
-      <img class="reading-list-item--cover" [src]="b.coverUrl" />
+      <img class="reading-list-item--cover" [src]="book.coverUrl"  alt="Book Cover"/>
```

Comment:
For non-interactive element, we should keep ```alt=""```

Reply:
changed to alt=""